### ISSUE TITLE
Is this an online learning algorithm?

### ISSUE THREAD

**Issue by MaryleenAmaizu** [sshh12/Webcam-Anomaly-Detection#2]
Would you say with the way you have done the training that this algorithm is an online learning algorithm where the model is continually updated with new batch of data points? That's what I'm trying to achieve with a different kind of dataset and model but don't know how to correctly implement the online learning concept. Any help will be appreciated.

**Comment by sshh12** [comment]
I wouldn't consider this an online learning algorithm since the training is done in one shot. When the FTP server is in training mode it's simply saving everything for training later.

**Comment by sshh12** [comment]
For my code tweaking this:

https://github.com/sshh12/Webcam-Anomaly-Detection/blob/589370cd0163d682d1882d90e1e7b58b0e93507f/anomaly_server/__init__.py#L146-L149

to maybe prompt the user to manually verify if the prediction isn't confident and then modify:

https://github.com/sshh12/Webcam-Anomaly-Detection/blob/589370cd0163d682d1882d90e1e7b58b0e93507f/anomaly_server/image_tools.py#L56-L58

to something like
```python 
anomaly_model.fit(np.array([dimage]), np.array([manual_label])) # maybe save as well
return manual_label == 1
```
to train the model while running.
